Source/ROSIntegrationVision/Private/VisionComponent.cpp

```diff
@@ -56,6 +56,7 @@ ColorsUsed(0)
         Color->CaptureSource = ESceneCaptureSource::SCS_FinalColorLDR;
         Color->TextureTarget = CreateDefaultSubobject<UTextureRenderTarget2D>(TEXT("ColorTarget"));
         Color->TextureTarget->InitAutoFormat(Width, Height);
+		Color->TextureTarget->TargetGamma = 2.3f;
```

Comment:
Minor detail, but would you be able to fix the formatting here so that the lines match up?